content/en/docs/collector/configuration.md

```diff
@@ -174,25 +179,20 @@ service:
 
 ## Receivers <img width="35" class="img-initial" src="/img/logos/32x32/Receivers.svg"> {#receivers}
 
-A receiver, which can be push or pull based, is how data gets into the
-Collector. Receivers may support one or more
-[data sources](/docs/concepts/signals/).
-
-The `receivers:` section is how receivers are configured. Many receivers come
-with default settings so simply specifying the name of the receiver is enough to
-configure it (for example, `zipkin:`). If configuration is required or a user
-wants to change the default configuration then such configuration must be
-defined in this section. Configuration parameters specified for which the
-receiver provides a default configuration are overridden.
+Receivers collect telemetry from one or more sources. They can be pull or push
+based, and may support one or more [data sources](/docs/concepts/signals/).
 
-> Configuring a receiver does not enable it. Receivers are enabled via pipelines
-> within the [service](#service) section.
+Receivers are configured in the `receivers:` section. Many receivers come with
```

Comment:
I'd like to suggest that we drop the colon from the section name. The colon is an artifact of the YAML syntax. The section name is just the YAML key identifier. WDYT?

```suggestion
Receivers are configured in the `receivers` section. Many receivers come with
```

If you agree, this change will need to be applied elsewhere in the page.